src/views/List.jsx

```diff
@@ -1,8 +1,10 @@
+import { Link } from 'react-router-dom';
 import { useState } from 'react';
 import { ListItem } from '../components/ListItem';
 
-export function List({ data }) {
+export function List({ data, listPath, loading }) {
 	const [search, setSearch] = useState('');
+	const [listName] = useState(listPath.split('/')[1]);
```

Comment:
I think this one slipped by in the most recent commit 😄, `const listName = listPath.split('/')[1] ` can replace listName as a state varaiable.

Reply:
Thanks for the note on this. I reviewed a few Stack Overflow posts where developers expressed concerns regarding efficiency and reliability pertaining to re-rendering when using standard variables. I'm going to leave it as a state variable based on that.